### DISCUSSION TITLE
Suppress N+1 Query alerts

### DISCUSSION THREAD

**Discussion by wpearse** [getsentry/sentry-php#1548] [Q&A]
Hello! 👋 

We love the new N+1 query alerts, they've helped us easily identify a few performance issues in our PHP/Laravel codebase. 

We've resolved ~80% of these alerts, and there are ~20% left that we don't intend to resolve until later in the year until we're working on those areas of the code again. (A lot of the offending code will be replaced.)

We have implemented the `before_send` callback with the intention of identifying these N+1 Query alerts, and dropping them before they're sent to Sentry but we're having trouble correctly identifying an event that is an N+1 Query, so that we can suppress it. Any tips?

Thanks in advance 🙏

**Comment by stayallive** [comment]
Hey @wpearse, the N+1 queries are not actually produced by the SDK but by Sentry itself so the `before_send` is not going to help! 

It is a really good question though... I am going to ping @cleptric to see if there is a better way possibly but my first guess would be to "Ignore" them from the Sentry UI which should make them move to the "Ignored" tab and not re-appear since they are not resolved and be silenced that way.

**Reply by cleptric** [reply]
Right, you should ignore the performance issues you are not interested in. Performance issues do not count against your errors quota, so it's more of a concern of triaging them and cleaning up your issues stream.
